docs/gettingstarted/sc4snmp-installation.md

```diff
@@ -12,24 +12,16 @@ microk8s helm3 search repo snmp
 Example output:
 ``` 
 NAME                                  	CHART VERSION	APP VERSION	DESCRIPTION                     
-splunk-connect-for-snmp/snmp-installer	0.1.1        	1.16.0     	A Helm chart for Splunk for SNMP
+splunk-connect-for-snmp/splunk-connect-for-snmp	0.1.1        	1.16.0     	A Helm chart for Splunk for SNMP
 ```
 
 ### Download and modify deployment_values.yaml and config_values.yaml files
 ```
-curl -o ~/deployment_values.yaml https://raw.githubusercontent.com/splunk/splunk-connect-for-snmp/develop/deploy-helm/snmp-installer/deployment_values.yaml
-curl -o ~/config_values.yaml https://raw.githubusercontent.com/splunk/splunk-connect-for-snmp/develop/deploy-helm/snmp-installer/config_values.yaml
-curl -o ~/static_values.yaml https://raw.githubusercontent.com/splunk/splunk-connect-for-snmp/develop/deploy-helm/snmp-installer/static_values.yaml
+curl -o /values.yaml https://raw.githubusercontent.com/splunk/splunk-connect-for-snmp/develop/charts/values.yaml.example
```

Comment:
If we have only one values file `values.yaml` we can use this command instead:
`microk8s helm3 inspect values splunk-connect-for-snmp/splunk-connect-for-snmp > values.yaml
`
I see `values.yaml` is different from `values.yaml.example`, shouldn't we allow users to see both? 
Also, is putting `values.yaml` to root directory `/` intended?